File: c/get-age.c
/**
 * Tanggal: 5 Maret 2026
 * Link: https://www.codewars.com/kata/557cd6882bfa3c8a9f0000c1/train/c
 */

#include <stdio.h>

int get_age(const char *input_string) {
    return (int)(input_string[0]) - 48;
}

int main() {
    printf("%d\n", get_age("5 years old"));
    printf("%d\n", get_age("1 years old"));
    printf("%d\n", get_age("9 years old"));

    return 0;
}

File: php/get-age.php
<?php
# Tanggal: 4 Juli 2026
# Link: https://www.codewars.com/kata/557cd6882bfa3c8a9f0000c1/train/php

function get_age($response) {
    return intval($response[0]);
}

echo get_age("3 years old") . PHP_EOL;
